StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Components/NavigationBar/LinkSheetNavigationBar.swift

```diff
@@ -144,26 +145,13 @@ class LinkSheetNavigationBar: SheetNavigationBar {
         button.accessibilityLabel = accessibilityLabel
         button.accessibilityIdentifier = accessibilityIdentifier
 
-        // Set fixed size for the button
-        button.translatesAutoresizingMaskIntoConstraints = false
-
         // Constrain the button size
+        button.translatesAutoresizingMaskIntoConstraints = false
         NSLayoutConstraint.activate([
             button.widthAnchor.constraint(equalToConstant: size),
             button.heightAnchor.constraint(equalToConstant: size),
         ])
 
-        // Constrain the image view size
-        if let imageView = button.imageView {
```

Comment:
FYI this is generally a no-no, you shouldn't modify another view's subviews constraints